sdk/formrecognizer/Azure.AI.FormRecognizer/tests/SessionRecords/FormRecognizerClientLiveTests/FormRecognizerClientCanAuthenticateWithTokenCredential.json

```diff
@@ -1,118 +1,147 @@
2/sDxBpf2T/AEiW3sL\u002B3/49L\u002BKvTf8Agmf\u002B0DoH7N/7Ytn4p8QXf2PR/wCyLmw8\u002BeDzvI/1cn/tOuc9Q\u002Bpv2Vv\u002BCQPir9m79rPwl401TxXpN3pHhKe5nP2eD/Sz\u002B7kjj/8ARlfoPcXH9pVxH7P/AO0h4V/a0\u002BG//CVeFbr7Zaf8\u002B/8Ay2t/\u002BuldRXQeXVPG/wDgqQbXTP8Agmd8UP8Arwig/wC/ksdfC/8Awb32H9pftaeLtV/6BHhn/wBGS19vf8FZbcan/wAE0fif9ltvtQ8i1uCfT97FX56/8En/ANtjwX\u002Bxz8WtX/4Sq1vP7I8RaR9n\u002B0WFv9s/e\u002Bb5lB2UT7T/AOC\u002BhbUf2BftVx96016yH/XD95XmX/BszNaqPi1a/wB6\u002BsSP\u002BePU1n/8FZP\u002BCkXwh/af/Y7/AOEL8K6teat4i1e/tp4IPs7w/ZxHJ5h\u002B0Ve/4Np9Y/4qT4uWv/L5/oP/ALVoIPhr9oJ93/BQ3xZ/09fETH/k1Gf61/QHqFva6n4b/sq6tf8AQ/s/2f7P/wBsq/n/APjBb2vhv9vzWPtV19js9I\u002BIv2i/uPs//LL7d/ra/aj/AIb4\u002BBf/AB9f8LK8Of8AHv8A8e/2j99/qqC6p\u002BLX7OpHgj/gph4T\u002BX7LZ2vxEkt7cwfwx/apI/L/AKfjX2T/AMHMFv8A8Vr8I/8Ar3vv/af/AMbr4r\u002BEMA8Tft1\u002BHbrSbn7VZ6t8TBPBP\u002B887y/t3meb5f8A1zr7Q/4OX58/Fr4Sf9e\u002Bpf8AtKgD1z/ghx4H8P6l\u002Bwr/AGrdaV9svNX1\u002B5\u002B0efb/APTX/wC1181/8HEHhDS/DXxr\u002BHY0rS7Sz\u002B2aTc/8e/8A2zr6a/4IcfEbwqv7C2j\u002BH7zX9Js9Xs9Wvbj7PcTxwzf62SvnH/g4A8UaT42\u002BNXw7tdK1XSdWOk6Rc/b/ALPced9n8zy/L/8AalAH15/wST1m58Sf8E4/A91cZu7nN1bjPYR/u/8A2nXv1fOn/BGf/lG/4Rtf\u002Bn\u002B5/wDRtfRP2mtDjrHwp/wcTzf2f\u002ByZ8Ov\u002Bxm/9tZa9Y/4Is/sv\u002BH/gn\u002BzhpHir7L9s8RfESwjv7/z9n\u002Bqk/wCWVeMf8HGX/Junw4/0nd/xOJOPX/RpK\u002Bvv\u002BCf9x/xgr8I/\u002BwBbf\u002BiqzNv\u002BXR\u002BRn/BYvwt/whP7dvxOtbW322l5Da3\u002BfTzI\u002Btfsf\u002Byvcf2l\u002ByX8L7r/AKlmx/8ARVfkn/wXgnz\u002B354i/wCXP/iQW3/bx/rK/Wz9k\u002B4tdS/ZK\u002BF/2X/jz/sGxt//ACFQbH5Y/wDBWjwrdfGz/grvaeFt32M6v9i0/wA//rp5lfol8Of\u002BCfHgH4c/smXngHVtKtLz/QJftFx\u002B7/0j91/0zr4T/bh03\u002B0f\u002BC8fhC19bjTZ/wD0ZJX606x01f8A68Jf/RVAH89n7KzweCP21/A/2RbXFl40isYfoZZIx/MV\u002B/HjD/kJXlfgT\u002Bz5cfYP2yPCJuPujx2uf/AmQH9a/fDxxcf8TKgKxzM9VLj\u002BtLqPSoaDMuVJVT7R9al\u002B00ATUUUUAFFSVHQAVJUdFAFu3/pXMfEC35s66K2rnfiR1s65zQ5m5qbR7f8A4mVU60dH/pWZ0HT6R0roLfrXM\u002BF/tVdDp9vQBdptxb/2bRc06gCTz6jpv2mj7TQA6iiig5yS36UefRR5FAEv2j60lFFaAWKtaPb1Db3FTW1AGhc1Nb29R037TQBNcXFFU/tFH2igC5VS4uKl\u002B01DQAVYqvRQBYoqvR9ooALjrRUX2j6VLQAUUfaKjoAko\u002B0VHRQBJ9oqL7R9KPtH0paDMkpbe4qKigCSluLioqK6ACk\u002B0fSlornNBPtH0paT7R9KgroMyxRVeiuc0J/tH0par0UGZPb/ANKWiq9ADrmj7TTaKDQKn\u002B0fSovtNNoAdc1Dc6hUlR10GYfabWiobapq5wJvtNFQ0V0AZ9v0oqSo6ACpKjt\u002BlSUHOFTW1Q1YoOgb9pqaesnUNQ/s2of7Q/tKgDYqSofs1TT0HOFSVQt9Qq3QBJRRUdB0ElWKr077TQBNRVerlBzkVv8A0qWiqtxcf8\u002BtAFqj7PRVig6CC3t6nptzTqDnJKKhtqmoAybi4rI1D71a9/WTcXFAGfUlx1qK3/pVu5qawUSrRSfZ/pS1idgUUUUAJ9n\u002BlFv/AEpakoAKLjrRUdAElFFFABRRR9noAKjoooAkt\u002BtTW9xVWkt7igD8lv8AgrD4Xuv\u002BHhXiG6urW7\u002By6vY239k3H/LG48uIeZX6e/8ABN/xhpfxI/Y58I2nh\u002B6\u002B2Xnh23\u002BwX/8A073P/LTy/M/55V8lf8FwPhPdeJPBPhHxpa2v/Io3Etvf/wDTC2uP\u002Ben/AH7r87NN8Y\u002BINR0670vStU1b7J/y8fYJ5P8A2nQdB6L/AMFHdP8A7S/b3\u002BMd1am1/wCQqYfPM8fkiP7PH0/6acfnmv2q/Zv1D\u002B0v2b/hzdf9QCx/9FV\u002BGP7O/wCzP4g/aQ8bf8I/4f0q8vP9Pi\u002B33H/LG3/66V\u002B7Hg/R7X4b\u002BCfDvh\u002B1/wCPPw7YRWFv/wBs62/5dnOflj4x8SaX8Of\u002BC4Hi3xR4gu/slno\u002BvxX
```

Comment:
Do we want filestreams logged?  (I'm assuming that's what this is)  I thought @kinelski had done some work to add these at runtime.

Reply:
plop. will undo